Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2_test.py

```diff
@@ -704,6 +704,35 @@ def test_remote_script_results(mocker, requests_mock):
     assert outputs[0].get("downloadUrl") == "https://url/1"
 
 
+def test_get_power_query_results(mocker, requests_mock):
+    """
+    Given
+        - required query, from_date and to_date arguments
+    When
+        - running sentinelone-get-power-query-results command
+    Then
+        - returns a table of result if data present
+    """
+    json_output = util_load_json('test_data/get_power_query_response.json')
+    requests_mock.get("https://usea1.sentinelone.net/web/api/v2.1/dv/events/pq-ping",
+                      json=json_output)
+    mocker.patch.object(demisto, 'params', return_value={'token': 'token',
+                                                         'url': 'https://usea1.sentinelone.net',
+                                                         'api_version': '2.1'})
+    mocker.patch.object(demisto, 'command', return_value='sentinelone-get-power-query-results')
+    mocker.patch.object(demisto, 'args', return_value={
```

Comment:
Its worth adding some cases with missing args, such as a case without the query_id and a case with accounts_id and query (In this case what happens? can it co-exists?)